<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>206</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>100</width>
    <height>10</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
color: black;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>500</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <widget class="QWidget" name="widget_3" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-10</y>
         <width>288</width>
         <height>401</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>288</width>
          <height>71</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">
background-color:#030bff

</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>29</y>
           <width>271</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:#02ebfe </string>
         </property>
         <property name="text">
          <string>Chinh sua xe</string>
         </property>
        </widget>
       </widget>
       <widget class="QLineEdit" name="ten">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>90</y>
          <width>113</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#98e6ff</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>ten xe</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="hang">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>90</y>
          <width>113</width>
          <height>16</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>1000</width>
          <height>1000</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#98e6ff</string>
        </property>
        <property name="placeholderText">
         <string>hang xe</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="namsanxuat">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>120</y>
          <width>113</width>
          <height>16</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>1000</width>
          <height>1000</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#98e6ff</string>
        </property>
        <property name="placeholderText">
         <string>nam san xuat</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="hinh">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>140</y>
          <width>113</width>
          <height>16</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>1000</width>
          <height>1000</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#98e6ff</string>
        </property>
        <property name="placeholderText">
         <string>link anh</string>
        </property>
       </widget>
       <widget class="QComboBox" name="loai">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>120</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#98e6ff</string>
        </property>
        <item>
         <property name="text">
          <string>Tay ga</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Con tay</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Xe so</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Xe dien</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="layanh_btn">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>140</y>
          <width>56</width>
          <height>16</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#98e6ff</string>
        </property>
        <property name="text">
         <string>lay anh</string>
        </property>
       </widget>
       <widget class="QPushButton" name="save_btn">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>180</y>
          <width>56</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Lưu</string>
        </property>
       </widget>
       <widget class="QPushButton" name="cancel_btn">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>180</y>
          <width>56</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Hủy</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
